admin-dev/themes/default/scss/controllers/_carrier-wizard.scss

```diff
@@ -15,12 +15,13 @@
   .stepContainer {
     position: relative;
     display: block;
+    height: 100% !important;
```

Comment:
can you tell why do why do we need to verride this with `!important` ?

It's usually a bad practice, but if there's no other choice well..

Reply:
Because a `fadein` is done and there are float blocks. So the `fadein` resize with the bad height.

Reply:
Did you check the JS inside smarty templates? I think we can get rid of the height calculation by modifying the JS inside these templates, I remember working on such a thing